radio/src/targets/pl18/board.h

```diff
@@ -101,14 +132,22 @@ extern "C" void SDRAM_Init();
 #define NUM_TRIMS                       8
 #define DEFAULT_STICK_DEADZONE          2
 
-#define BATTERY_WARN                  37 // 3.7V
-#define BATTERY_MIN                   35 // 3.4V
-#define BATTERY_MAX                   43 // 4.3V
+#if defined(RADIO_NV14_FAMILY)
+  #define BATTERY_WARN                  36 // 3.6V
+  #define BATTERY_MIN                   35 // 3.5V
+  #define BATTERY_MAX                   42 // 4.2V
+#else
+  #define BATTERY_WARN                  37 // 3.7V
+  #define BATTERY_MIN                   35 // 3.4V
+  #define BATTERY_MAX                   43 // 4.3V
```

Comment:
This is because the non-NV14 have high voltage batteries? Maybe a comment? ;)

Reply:
Just copy from nv14 target.